packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -300,7 +300,7 @@ gmaps.MarkerOptions _markerOptionsFromMarker(
     ..visible = marker.visible
     ..opacity = marker.alpha
     ..draggable = marker.draggable
-    ..icon = _gmIconFromBitmapDescriptor(marker.icon);
+    ..icon = _gmIconFromBitmapDescriptor(marker.icon!);
```

Comment:
we are using icon before null-checking it here... but it seems like this is from google_maps_flutter_web, so we don't need to care about it, right?

Reply:
imo it's always good to care about this (even tho we're not using it)